### DISCUSSION TITLE
Issue cleanup

### DISCUSSION THREAD

**Discussion by jacadzaca** [collective/icalendar#392] [Ideas]
Hey there. Going through the repository's issues can be quite cumbersome due to some of them not being 'real' issues, being already solved etc. To help with that, I put together a list of issues I think could be closed (reading the comments reveals that etc.) 

I'd also like to propose adding 2 new tags: `good first issue` and `help wanted`, so new contributors have easier time finding issues they'd like to work on. We could also list the project on [GoodFirstIssue](https://goodfirstissue.dev/) or the likes.

The list:
## Solved (or seemingly solved)
  - #348
  - #335
  - #332
  - #326
  - #323
  - #300
  - #298
  - #295
  - #291
  - #290 (see #295?)
  - #288 (site's up)
  - #285 (no more extras in [tox.ini](https://raw.githubusercontent.com/collective/icalendar/master/tox.ini))
  - #276
  - #275 (calendar.walk exists)
  - #270 (see #288)
  - #264 (see #270)
  - #263
  - #254
  - #166 (not a bug)
  - #135 (we use codecov)
  - #126
  - #121 ("""solved""")
  - #104
  - #84

## Out of scope of the project
  - #341
  - #317
  - #162
  - #85 (not really an issues?)

# Issues that should be given the:
## good first issue tag
  - #350
  - #286
  - #268

## help wanted tags
  - #351
  - #350
  - #349
  - #286
  - #268
  - #266 and #130
  - #215
  - #69

**Comment by geier** [comment]
Thank you for the effort!

I agree with all of your suggestions. 

New labels are created and attached.

**Reply by geier** [reply]
@niccokunzmann  I'd also be in favor of closing solved issues that @jacadzaca has identified. Perhaps we can move the out of scopes one to dicsussions?

**Reply by niccokunzmann** [reply]
> Perhaps we can move the out of scopes one to discussions?

That sounds good... Personally, to keep the discussion count low, I would say it can be closed and suggest that is can be recreated as a discussion.

> Thanks for creating this issue. I have identified that this is out of scope for this module but still other people might be interested to talk about it and so I suggest that you create a discussion about this here: https://github.com/collective/icalendar/discussions. I will close this issue now and please link the new discussion you create as a comment, so people can have a look!

**Comment by niccokunzmann** [comment]
@geier I added @jacadzaca with write access. This can speed up development and take a load from our shoulders.
@jacadzaca You have write access now. There seems to be no disagreement with the changes you proposed and you can go ahead and do something about them. Please get back to us if you have any questions. You can merge and review pull requests but please do not merge your own pull requests. Happy Hacking!

**Comment by jacadzaca** [comment]
I closed most of the issues I've listed, except for two. #348 might require a better fix than the one proposed and #298  is a bug, which I have no idea how to fix. I assigned @geier to help with #348 and #298 is labeled `help wanted`

**Reply by jacadzaca** [reply]
also, #162 should be closed when we remove pytz.

**Comment by jacadzaca** [comment]
[GoodFirstIssue](https://goodfirstissue.dev/) pull request - https://github.com/deepsourcelabs/good-first-issue/pull/461
